uyuni-docs-helper

```diff
@@ -34,6 +34,18 @@ GITREPO='https://github.com/uyuni-project/uyuni-docs.git'
 # Default Git reference
 GITREF='master'
 
+# use gnu-getopt on macOS
+getopt_for_macos() {
+  if [ $(uname -s) == 'Darwin' ]; then
```

Comment:
```suggestion
  if [ "$(uname -s)" == 'Darwin' ]; then
    if [ "${SHELL}" == "/bin/fish" ]; then
      print_error "fish shell is not supported, please use bash or zsh for now"
      exit 1
    fi
```
It's better to add quotes to `uname -s` as otherwise it will break if for some reason `uname -s` returns a string with more than one word.

And since you mention `fish` appears to be broken, maybe we can help the user understand what's going on (review what's the value of `SHELL` if using fish, I am not sure if I am using the right path :-))

Reply:
I don't know if there are other shells that can be broken in MacOS. Otherwise you can check if the shell is bash or zsh and show an error for all others.

Reply:
I use fish, but was not able to figure out what to change there to make it work under macOS, yet. I am somehow still new to fish.

Reply:
For this PR, it's OK.

If you want support for fish, make sure you create a new issue, and we can try to figure out what's broken.